mythril/laser/ethereum/state/account.py

```diff
@@ -85,6 +89,27 @@ def __str__(self) -> str:
         # TODO: Do something better here
         return str(self.printable_storage)
 
+    def merge_storage(self, storage: "Storage", path_condition: Bool):
```

Comment:
I think we should move this merging specific logic to the plugin? 
(Either have a Storage subclass that is dependency injected into Laser, or have the function as a helper there)